tracsubtickets/checker.py

```diff
@@ -89,7 +88,8 @@ def main(args=sys.argv[1:]):
 
     # check all the environments
     for env in envs:
-        check_subtickets(env)
+        with env.db_transaction as db:
+            check_subtickets(db)
 
```

Comment:
Will do this at a later time. checker.py seems not to work at all and it is just a debugging tool.